github/github.go

```diff
@@ -91,6 +91,9 @@ const (
 
 	// https://developer.github.com/changes/2016-09-14-Integrations-Early-Access/
 	mediaTypeIntegrationPreview = "application/vnd.github.machine-man-preview+json"
+
+	// https://developer.github.com/changes/2016-12-14-reviews-api/
+	mediaTypePullRequestReview = "application/vnd.github.black-cat-preview+json"
```

Comment:
I know it's a mouthful, but, there should be a `Preview` stuffix at the end. These are all preview APIs, this one happens to be named "Pull Request Reviews". So:

```Go
mediaTypePullRequestReviewsPreview = ...
```